packages/phrases/src/locales/zh-cn.ts

```diff
@@ -40,6 +40,13 @@ const translation = {
       subtitle:
         'Setup a mobile, single page or traditional application to use Logto for authentication.',
       create: 'Create Application',
+      application_name: 'Application Name',
+      client_id: 'Client ID',
+      type: {
+        native: 'Native App',
+        spa: 'Single Page App',
+        tranditional: 'Tranditional Web App',
+      },
```

Comment:
We may need an issue to track this i18n feature.

Reply:
we can replace this at once

Reply:
added LOG-1822